src/components/Dropdown/DropdownComponent.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div v-if="isClickable" class="dropdownclick" v-click-away="closedropdown">
+    <button href="#" class="dropbtnclick" @click="toggleVisibility">
+      {{ Heading }}
+    </button>
+    <div ref="dropdown" :class="[isVisible ? isActive : dropdowncontent]">
+      <a
+        v-for="(item, index) in content"
+        v-bind:key="index"
+        :href="item.href"
+        @click="click"
+      >
+        {{ item.option }}
+      </a>
+    </div>
+  </div>
+
+  <div v-else class="dropdown">
+    <button href="#" class="dropbtn">
+      {{ Heading }}
+    </button>
+    <div v-if="isVisible" ref="dropdown" class="dropdown-content">
+      <a
+        v-for="(item, index) in content"
+        v-bind:key="index"
+        :href="item.href"
+        @click="click"
+      >
+        {{ item.option }}
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+import { ref } from "vue";
+
+export default {
+  props: {
+    Heading: String,
+    content: Array,
+    href: String,
+    isClickable: Boolean,
+  },
+  setup(props) {
+    const isVisible = ref(true);
```

Comment:
you can keep the default as false, closed.

Reply:
it should be true by default according to functionality, for hover to work upon :)

Reply:
oh! it was open for click trigger in storybook though, so maybe we can keep the initial value to true/false according to the current mode?